avaframe/ana4Stats/probAna.py

```diff
@@ -261,8 +263,14 @@ def probAnalysis(avaDir, cfg, module, parametersDict='', inputDir=''):
 
                 # Load data
                 fileName = peakFilesDF['files'][m]
-                data = np.loadtxt(fileName, skiprows=6)
                 dataLim = np.zeros((nRows, nCols))
+                fileData = IOf.readRaster(fileName)
+                data = np.flipud(fileData['rasterData'])
+
+                # fetch contourline info
+                xGrid, yGrid, _, _ = gT.makeCoordGridFromHeader(fileData['header'])
+                x, y = pU.fetchContourCoords(xGrid, yGrid, data, float(cfg['GENERAL']['peakLim']))
+                contourDict[fileName.stem] = {'x': x, 'y': y}
```

Comment:
Did you check whether this slows the function down? sometimes contour generation can take quite some time...

Reply:
I tried. since it is only on contour per sim, it is not slowing it down